apps/io-services-cms-webapp/src/reviewer/review-checker-handler.ts

```diff
@@ -106,44 +106,40 @@ export const buildIssueItemPairs =
  */
 export const updateReview =
   (dao: ServiceReviewDao, fsmLifecycleClient: ServiceLifecycle.FsmClient) =>
-  (data: TE.TaskEither<Error, IssueItemPair[]>): TE.TaskEither<Error, void> =>
+  (issueItemPairs: IssueItemPair[]): TE.TaskEither<Error, void> =>
     pipe(
```

Comment:
**suggestion (non-blocking):** use _flow_.
You could use _flow_ instead of _pipe(issueItemPairs_
```suggestion
    flow(
```

Reply:
In this case pipe cannot be replaced by flow as is the start of an arrow function